### ISSUE TITLE
Vue2 | Range issue: v-model does not working

### ISSUE THREAD

**Issue by K-KOYANAGI** [OnsenUI/OnsenUI#3042]
__Environment__

  
```
[Core]
  onsenui latest
  
[Framework]
  vue 2.7.1

[Framework binding]
 vue-onsenui legacy

[Platform]
  Desktop - OS X 10.15.7 64-bit

[Browser]
  Desktop - Chrome 108.0.0.0 on OS X 10.15.7 64-bit
```

__Encountered problem__
  
The bound variable does not change by the slider value changes.

__How to reproduce__
  
`volume` is always `25` (initial value) in this demo.

  
  [__Demo link__](https://onsen.io/playground/?issue)

  

- __HTML__

```html
<template id="main">
  <v-ons-page>
    <v-ons-toolbar>
      <div class="center">Range</div>
    </v-ons-toolbar>

    <v-ons-list>
      <v-ons-list-item>
        Adjust the volume:
        <v-ons-row>
          <v-ons-col width="40px" style="text-align: center; line-height: 31px;">
            <v-ons-icon icon="md-volume-down"></v-ons-icon>
          </v-ons-col>
          <v-ons-col>

            <v-ons-range v-model="volume" style="width: 100%;"></v-ons-range>

          </v-ons-col>
          <v-ons-col width="40px" style="text-align: center; line-height: 31px;">
            <v-ons-icon icon="md-volume-up"></v-ons-icon>
          </v-ons-col>
        </v-ons-row>
        Volume: {{ volume }} <span v-show="Number(volume) > 80">&nbsp;(careful, that's loud)</span>
      </v-ons-list-item>
    </v-ons-list>
  </v-ons-page>
</template>

<div id="app"></div>
```

- __JS__

```javascript
new Vue({
  el: '#app',
  template: '#main',
  data() {
    return {
      volume: 25
    };
  }
});
```

**Comment by K-KOYANAGI** [comment]
I noticed that this problem only occurs in `"vue-onsenui": "2.7.1"`. By downgrading `vue-onsenui` to `2.7.0` it was fixed.

**Comment by as-hirose** [comment]
@K-KOYANAGI  Hi. We have released vue-onsenui v2.7.2 and fixed this issue.
Thank you for your contributions.

**Comment by K-KOYANAGI** [comment]
I'm sorry for the late response.
Thanks for fixing this problem.
